### DISCUSSION TITLE
Suggestion: New mod simillar to SponsorBlock, but with the most important parts of video

### DISCUSSION THREAD

**Discussion by bablabl** [ReVanced/revanced-discussions#575] [Question] [answered]
Hi

Youtube Vanced and ReVanced are already amazing with all of features u guys added into it, but what i think isn't hard to make is mod simillar to SponsorBlock, but instead of users  selecting sponsor-part of video, they could select the most important part of it, for example:

REVIEW OF MICROPHONE: Whole video is 12 minutes long, but you are now able to click twice onto thicken part of timeline (or any other gesture that creators of mod would choose) and it will skip useless parts of video.

I think it would be very useful and possible to create, because SponsorBlock community is already huge, and there's really thin line between marking sponsor-parts and useless parts of video, so it's not a big deal for already existing community to select the most important parts of video

**Comment by oSumAtrIX** [comment, marked as answer]
Marking the highlight of the video is already possible in SponsorBlock.

**Reply by bablabl** [reply]
O, then im blind

**Reply by bablabl** [reply]
Is it in new ReVanced versions of Youtube? I can't see it in my YOutube Vanced (works better on my phone)

**Reply by bablabl** [reply]
Yeah sadly i don;t think its on latest Vanced version

**Comment by oSumAtrIX** [comment]
RE: https://github.com/revanced/revanced-patches/issues/441

**Reply by bablabl** [reply]
alr man, thanks i hope they will see this, gn
